app/services/normalise-charge.js

```diff
@@ -1,13 +1,13 @@
 'use strict'
 
 // NPM dependencies
-const humps = require('humps')
 const lodash = require('lodash')
 
 // Local dependencies
 const countries = require('../services/countries')
 const normaliseCards = require('../services/normalise-cards')
 const userIpAddress = require('../utils/user-ip-address')
+const keyCamelizer = require('../utils/key-camelizer')
```

Comment:
purely a style choice but `camelize` is a named export and could be required via destructuring 

```suggestion
const { camelize } = require('../utils/key-camelizer')
```